functions.php

```diff
@@ -280,4 +279,72 @@ function validateFormSignUp(mysqli $link, array $registration): array {
     return $errors;
 }
 
+function getUserDb(mysqli $link, string $email): mixed {
+	$sql = "SELECT * FROM user WHERE email = ?";
+    $stmt = mysqli_prepare($link, $sql);
+    mysqli_stmt_bind_param($stmt, 's', $email);
+    mysqli_stmt_execute($stmt);
+    $result = mysqli_stmt_get_result($stmt);
+    $user = mysqli_fetch_array($result, MYSQLI_ASSOC);
+
+    return $result && !empty($user) ? $user : null;
```

Comment:
```suggestion
    return $result && !empty($user) ? $user :[];
```
и тип не mixed, а array.
mixed старайся не использовать лишний раз